addons/web/static/src/js/view_list.js

```diff
@@ -2298,7 +2298,7 @@ instance.web.list.Binary = instance.web.list.Column.extend({
         var text = _t("Download");
         var value = row_data[this.id].value;
         var download_url;
-        if (value && value.substr(0, 10).indexOf(' ') == -1) {
+        if (value && value.substr(0, 10).indexOf(' ') != -1) {
```

Comment:
> if value is valid base64, created a inline link, user downloads that link was not show valid file name. this is the real probleam

OK but the change is not correct, if the value is bin_size formatted it will now generate a link `data:application/octet-stream;base64,64 KB` which is complete nonsense. And in that case there's no sense in ever downloading the raw data so the list view should be changed to always fetch its records with bin_size set. Either way, that should be part of a different pull request, it has nothing to do with the other fix.